document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/handler/accounts/AccountsHandlerImpl.java

```diff
@@ -48,13 +60,19 @@ public DocumentInfoResponse getAbridgedAccountsData(Transaction transaction, Str
         try {
             AbridgedAccountsApi abridgedAccountData = accountsService.getAbridgedAccounts(abridgedAccountLink);
 
-            return createResponse(transaction, accountType, abridgedAccountData);
+            return createResponse(transaction, accountType, abridgedAccountData, mimeType);
         } catch (ServiceException e) {
             Map<String, Object> logMap = new HashMap<>();
             logMap.put("resource", abridgedAccountLink);
             logMap.put("accountType", accountType);
             LOG.error("Error in service layer", logMap);
             throw new HandlerException(e.getMessage(), e.getCause());
+        } catch (ParseException e) {
+            Map<String, Object> logMap = new HashMap<>();
+            logMap.put("resource", abridgedAccountLink);
+            logMap.put("accountType", accountType);
```

Comment:
Can you use constants instead of duplicating the logging keys please?

Reply:
I've added the exception into the log